.github/pull_request_template.md

```diff
@@ -18,11 +19,12 @@ not fully covered by unit tests or manual testing can be complicated. -->
 
 ### Checklist
 <!-- Put an 'x' in all the boxes that apply -->
-- [ ] I submit my changes into the `develop` branch
-- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
+- [ ] I have added unit tests to cover my changes.​
+- [ ] I have added integration tests to cover my changes.​
+- [ ] I have added e2e tests for validation.
+- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/releases/1.1.0/CHANGELOG.md).​
 - [ ] I have updated the [documentation](
-  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
-- [ ] I have added tests to cover my changes
+  https://github.com/openvinotoolkit/datumaro/tree/releases/1.1.0/site) accordingly
```

Comment:
here also.

Reply:
We are able to just keep this file in `develop` branch, not `release` branch.

Reply:
agree.